deploy/src/index.ts

```diff
@@ -22,6 +22,51 @@ async function checkDeploy(name: string, namespace: string) {
     return toolrunner.exec();
 }
 
+async function kustomization(manifests: string[], image: string, tag: string){
+  const path = 'kustomization.yaml';
+  try{
+      await promises.access(path);
+      return;
+  } catch(err) {
+    if (err.code !== 'ENOENT') {
+      throw err;
+    }
+  }
+
+  const k = yaml.safeDump({
+    resources: manifests,
+    images: [{
+      name: image,
+      newName: image,
+      newTag: tag
+    }]});
+  console.log(k);
+  await promises.writeFile('kustomization.yaml', k);
```

Comment:
@rlamana is this `await` needed? Or will the call to `await kustomization(...)` ensure that this finishes?